packages/geo-diary-v2/src/scripts/webpack-dev.js

```diff
@@ -0,0 +1,152 @@
+const path = require('path')
```

Comment:
Think the scripts can be removed as the package.json references the root scripts folder. Again, something we should remove from the internal scaffold